lib/metasploit/framework/mssql/base.rb

```diff
@@ -0,0 +1,354 @@
+module Metasploit
```

Comment:
This could probably live in the rex proto namespace, but I'm not wanting to scope creep the original effort of just removing code duplication as a first pass